content/en/docs/hertz/getting-started/_index.md

```diff
@@ -1,49 +1,60 @@
 ---
-title: "Getting Started"
-linkTitle: "Getting Started"
+title: 'Getting Started'
+linkTitle: 'Getting Started'
```

Comment:
There is no need to change this

Reply:
https://github.com/cloudwego/cloudwego.github.io/blob/f64d0546cbbdfb84e74f756e15b5a19906f36065/.prettierrc.json#L3
This affects me... the prettier config